app/src/main/java/com/dd3boh/outertune/viewmodels/HomeViewModel.kt

```diff
@@ -145,6 +145,20 @@ class HomeViewModel @Inject constructor(
         isLoading.value = false
     }
 
+
+    fun loadMoreYouTubeItems(continuation: String?) {
+        if (isRefreshing.value || continuation == null) return
+
+        viewModelScope.launch(Dispatchers.IO) {
+            isRefreshing.value = true
+            val nextSections = YouTube.home(continuation).getOrNull() ?: return@launch
+            homePage.value = nextSections.copy(
+                homePage.value?.sections.orEmpty() + nextSections.sections
+            )
+            isRefreshing.value = false
```

Comment:
Not sure, if we should show the refresh indicator every time a new section is loaded?